resource-managers/yarn/src/main/scala/org/apache/spark/deploy/yarn/ResourceRequestHelper.scala

```diff
@@ -41,6 +42,9 @@ private object ResourceRequestHelper extends Logging {
   private val AMOUNT_AND_UNIT_REGEX = "([0-9]+)([A-Za-z]*)".r
   private val RESOURCE_INFO_CLASS = "org.apache.hadoop.yarn.api.records.ResourceInformation"
   private val RESOURCE_NOT_FOUND = "org.apache.hadoop.yarn.exceptions.ResourceNotFoundException"
+  private val APPLICATION_TIMEOUT_CLASS = "org.apache.hadoop.yarn.api.records.ApplicationTimeout"
```

Comment:
Resourcerequest helper doesn't seem like the right place for any of these changes since its not dealing with "resources".
I would rather just see this in Client class since its only used there